test/server.test.js

```diff
@@ -34,7 +34,7 @@ var FAST_CLIENT;
 var SERVER;
 
 if (SKIP_IP_V6) {
-    console.warning('IPv6 tests are skipped: No IPv6 network is available');
+    console.warn('IPv6 tests are skipped: No IPv6 network is available');
```

Comment:
@hekike :joy:

Reply:
This is why Travis was skipping the rest of the tests. Not sure why this wouldn't cause an error code though.

Reply:
ooups, VW